src/filter_noise.py

```diff
@@ -0,0 +1,35 @@
+from core_data_modules.logging import Logger
+from src.lib.pipeline_configuration import PipelineConfiguration
+
+log = Logger(__name__)
+
+
+class FilterNoise(object):
+    @staticmethod
+    def filter_noise_radio_messages(messages):
```

Comment:
This feels like a message filter and is probably better placed in MessageFilters

Reply:
Its applied after Apply manual codes. Unless I merge it in ApplyManualCodes.py